apps/hash-frontend/src/pages/[shortname]/entities/[entity-uuid].page.tsx

```diff
@@ -42,6 +44,15 @@ const Page: NextPageWithLayout = () => {
   const [draftEntitySubgraph, setDraftEntitySubgraph] =
     useState<Subgraph<SubgraphRootTypes["entity"]>>();
 
+  const entityFromDB =
+    entitySubgraphFromDB && getRoots(entitySubgraphFromDB)[0];
+
+  const entityOwnedById =
+    entityFromDB &&
+    extractOwnedByIdFromEntityId(entityFromDB.metadata.editionId.baseId);
```

Comment:
nit: casing (plus one on line 42 which I can't suggest on)
```suggestion
  const entityFromDb =
    entitySubgraphFromDb && getRoots(entitySubgraphFromDb)[0];

  const entityOwnedById =
    entityFromDb &&
    extractOwnedByIdFromEntityId(entityFromDb.metadata.editionId.baseId);
```